packages/shared/src/models/no-service-quadlet.ts

```diff
@@ -0,0 +1,23 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { BaseQuadlet } from './base-quadlet';
+
+export interface NoServiceQuadlet extends BaseQuadlet {
```

Comment:
suggestion(non-blocking): I have a preference for `ServiceLessQuadlet`, but that's a personal choice

Reply:
after some reflexion, I really prefer ServiceLessQuadlet